src/main/java/io/anserini/search/SearchHnswDenseVectors.java

```diff
@@ -16,6 +16,8 @@
 
```

Comment:
Can we remove this option in this class for now? We don't have a way to test it, and so I would rather _not_ have a feature than an _untested_ feature.